interfaces/builtin/fwupd.go

```diff
@@ -65,17 +65,26 @@ const fwupdPermanentSlotAppArmor = `
   # Allow fwupd to access system information
   /sys/devices/virtual/dmi/id/product_name r,
   /sys/devices/virtual/dmi/id/sys_vendor r,
+  /sys/firmware/dmi/tables/ r,
+  /sys/firmware/dmi/tables/** r,
```

Comment:
This is fine, but also provided by hardware-observe. I'm inclined to leave anything in /sys/firmware in this interface but letting the other things in /sys for hardware-observe (in other words, I think this one is ok to leave here).

Reply:
This is also removed now and replaced by hardware-observe as well! According this change, I add hardware-observe and shutdown as plugs for daemon and client separately, and snap works well to update bios. Great!